Tools/MGCB/SourceFileCollection.cs

```diff
@@ -21,9 +21,14 @@ public sealed class SourceFileCollection
         [XmlArrayItem("File")]
         public List<string> SourceFiles { get; set; }
 
+        [XmlArrayItem("File")]
+        public List<string> DestFiles { get; set; }
```

Comment:
Isn't this going to serialize both source and dest files together?  Isn't that bad?